pkg/account/component.go

```diff
@@ -262,8 +262,21 @@ func (c *component) UpdateLabelCredential(ctx context.Context, credentialID stri
 
 	//store the API call into the DB
 	// the error should be treated
-	additionalInfos, _ := json.Marshal(map[string]string{"credentialID": credentialID, "label": label})
-	_ = c.reportEvent(ctx, "SELF_UPDATE_CREDENTIAL", database.CtEventRealmName, currentRealm, database.CtEventUserID, userID, database.CtEventUsername, username, database.CtEventAdditionalInfo, string(additionalInfos))
+	additionalInfos, err := json.Marshal(map[string]string{"credentialID": credentialID, "label": label})
+	if err != nil {
```

Comment:
I think this change is not useful... Do we really think we can get an error here ? (but it is not a problem)